pandas/io/parsers.py

```diff
@@ -1582,6 +1581,7 @@ def __init__(self, f, **kwds):
         self.converters = kwds['converters']
 
         self.thousands = kwds['thousands']
+        self.decimal = kwds['decimal']
```

Comment:
@jreback I can't find any reference in `io.rst` to a particular engine when decimal option is used